tests/shell/shell_recorder/shell_recorder.sh

```diff
@@ -208,54 +208,54 @@ execute()
 
 run_script()
 {
-    while read line;
+    while read -r line;
     do
         OP=
         ARG=
-        cmd=$(echo $line|cut -d ' ' -f1)
+        cmd=$(echo "$line"|cut -d ' ' -f1)
         case "$cmd" in
             Mountpoint:)
-                Mountpoint=$(echo $line|cut -d ' ' -f2)
+                Mountpoint=$(echo "$line"|cut -d ' ' -f2)
                 ;;
             File:)
-                DBFile=$(echo $line|cut -d ' ' -f2)
+                DBFile=$(echo "$line"|cut -d ' ' -f2)
                 if [ "$DBFile" = "File:" -o -z "$DBFile" ]; then
```

Comment:
if [ "$DBFile" = "File:" -o -z "$DBFile" ]; then    
                              ^-- SC2166: Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.